Gemfile

```diff
@@ -40,6 +40,7 @@ end
 
 group :test, :development do
   gem 'awesome_print'
+  gem 'brakeman', '4.3.1'
```

Comment:
@aleksandarpetrushev why lock to this specific version?

Reply:
@damianlegawiec https://docs.codeclimate.com/docs/brakeman, says "Due to the acquisition of Brakeman, we're currently not able to upgrade our OSS Brakeman plugin to use Brakeman's latest version." This is the latest version it can work with codeclimate it seems